src/modules/partners/partner_service.ts

```diff
@@ -19,4 +19,22 @@ export namespace Partner {
 
     return result
   }
+
+  export const search = async (requestQuery: Entity.SuggestionRequestQuery): Promise<Entity.SuggestionResponse> => {
+    const partners = requestQuery?.name?.length >= 3 ? await Repository.search(requestQuery) : []
+
+    const data = partners.map(partner => ({
```

Comment:
I guess this doesn't need mapping anymore, because the return from the database returns an array of name and id, so you can just delete it

Reply:
I encountered a problem with the typing because I don't know the return type from Repository. Is there any way to cast the type to our desired type?

It says something like this 
![image](https://user-images.githubusercontent.com/52171676/140013337-190e2b2e-e3ab-48fb-9ded-29c8d8676717.png)